### PR TITLE
Fix/index creation

### PR DESCRIPTION
**Pull request by robert-bryson**
# Pull Request

Related to https://github.com/GSA/data.gov/issues/3648

## About

- Tries to integrate [@nickumia-reisys's solve](https://github.com/GSA/ckanext-geodatagov/pull/225) for the s3 upload issue
- Refactoring for organization (s3 globals, breaking in the tidy functions, etc)
- Lint other changes with `black`/`isort`

## PR TASKS

- [x] The actual code changes.
- [x] Tests written and passed.
- [x] Any changes to docs? [Yes](https://github.com/GSA/data.gov/wiki/CKAN-commands/ba3e427eacb4e28fb4dfe7ddc4eeec1210910a6f)
- [x] Bumped version number in [setup.py](https://github.com/GSA/ckanext-geodatagov/blob/1b1bad0b2ff06112e18c7f4f4fb1143baec1266a/setup.py#L13) (also checked on [PyPi](https://pypi.org/project/ckanext-geodatagov/#history)).

### CODE REVIEW

File: ckanext/geodatagov/cli.py [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
 from ckan.lib.search import rebuild
 from ckan.lib.search.common import make_connection
 from ckan.lib.search.index import NoopSearchIndex, PackageSearchIndex
 
 from ckanext.geodatagov.search import GeoPackageSearchQuery
 
-_INDICES = {
-    'package': PackageSearchIndex
-}
+_INDICES = {"package": PackageSearchIndex}
```
Comment (FuhuXia) [inline ckanext/geodatagov/cli.py:19]:
```suggestion
```

Comment (FuhuXia) [reply ckanext/geodatagov/cli.py:19]: defined twice?

**Review by FuhuXia — CHANGES_REQUESTED** [review]

**Comment by FuhuXia** [comment]
How big are the file uploads in total?

**Comment by robert-bryson** [comment]
> How big are the file uploads in total?

The sitemap files appear to be about 200kb each. The ~30k records on catalog dev result in about 6mb of files. I haven't run on prod, but obviously it will be a much larger number in total.

**Review by jbrown-xentity — APPROVED** [review]

**Review by FuhuXia — APPROVED** [review]

**Pull request merged**